.github/ISSUE_TEMPLATE/documentation-improvement.md

```diff
@@ -0,0 +1,22 @@
+---
+name: Documentation improvement
+about: Create a report to help us improve the documentation
+title: Documentation improvement
+labels: documentation
+assignees: danielolsen, rouille, BainanXia, jon-hagg, ahurli
+
+---
+
+### Problem
+Tell us about the confusion introduced in the documentation, what is lacking, etc.
+* I found [...] to be unclear because [...]
+* [...] made me think that [...] when really it should be [...]
+* There is no example showing how to do [...]
```

Comment:
Do you think we should further categorize the problems related to documentations, such as:
- Ambiguous
- Missing
- Outdated
- Wrong

Reply:
Done